src/style.css

```diff
@@ -0,0 +1,121 @@
+* {
+  padding: 0;
```

Comment:
bad idea to reset padding for all items in page. you can do this directly for a specific ones

Reply:
without this padding the tests will not pass